samples/display/cfb/README.rst

```diff
@@ -0,0 +1,35 @@
+.. _character_frame_buffer_sample:
+
+Character frame buffer
+######################
+
+Overview
+********
+
+This sample will display a vertical scrolling text string making use of the
```

Comment:
To me, "vertical scrolling text string" means it would look like this:

    V
    E
    R
    T
    I
    C
    A
    L

I think this sentence should instead be:

    This sample displays character strings using the Character Frame Buffer
    (CFB) subsystem framework.

I was hoping to find documentation for this CFB subsystem, but alas I can't find it.  Sigh.

Reply:
Point taken on the "vertical"

> I was hoping to find documentation for this CFB subsystem, but alas I can't find it. Sigh.

No there is not much of documentation on CFB but I did take some effort on documenting the sample in PR #11351 (which is till waiting for a review on the doc ;) )